lib/galaxy/datatypes/text.py

```diff
@@ -1421,3 +1422,11 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
             (lines[9].strip() == end_header and lines[10].strip() == grid_points)
             or (lines[9].strip() == end_header_spin and lines[10].strip() == grid_points_spin)
         )
+
+
+@build_sniff_from_prefix
+class Svif(Json):
+    """
+    Software Visualization Interchange Format (Satrio Adi Rukmono).
+    """
+    file_ext = "svif"
```

Comment:
Let's add the line ending since Git complains. This might crash on different server setups